### ISSUE TITLE
Dropped fly error + other out

### ISSUE THREAD

**Issue by AloisSeckar** [AloisSeckar/WBSC-Scoring#168]
Play should be connected:

![image](https://github.com/AloisSeckar/WBSC-Scoring/assets/7399922/8c065a3b-ed52-4b1a-ba77-d07c4c9e6fcd)

**Comment by AloisSeckar** [comment]
Might be resolved together with #180 ?

**Comment by lucafano04** [comment]
This is wrong, since an out has been made the error should be essented and the play at first shall be scored as O8

**Comment by AloisSeckar** [comment]
I copied the play from the WBSC manual. But maybe it got changed since. I will confirm with Linda Steijger.

**Comment by lucafano04** [comment]
> I copied the play from the WBSC manual. But maybe it got changed since. I will confirm with Linda Steijger.

The new WBSC manual is in a developing phase, fortunally there are the official rules at the moment plus some "national manuals" made by Anna Maria Paini witch is the "Scoring Director" of WBSC and president of CNC @ FIBS

**Comment by AloisSeckar** [comment]
Yea, I know Anna Maria, have been to their clinic in Prague in 2012 and also in San Jose, 2019. But the situation about the official manual is getting out of hands, if you ask me. In 2019 Anna Maria told us the manual is being finished and will be ready soon. Then nothing happened, just I got access to "draft" versions via Pablo Carpio. It is 2024 and still no development. It even already happened to me, that I was about to "correct" scorers in Barcelona, because they did something differently than I was used to, but they told me they were instructed to do that as this is the new way of doing it. It is hard to follow the development, when there is no solid reference to start from. Also maintainting this software would be difficult without a final set of features that we need to cover.

**Comment by lucafano04** [comment]
I know the situation, since i'm an official scorer in Italy and also a local responsable (Prov. autonoma di Trento) for team and official scorers i'm in the same situation, obviously scaled in a more local envrioment.

**Comment by AloisSeckar** [comment]
Answered by Dennis Duin / Linda Steijger from WBSC Europe:

_The example is still valid. As the text says, this is not a forced out. Had the ball been caught, there would have been a double play F8 – 85. Because the fly ball was missed, we have to write a decisive error for prolonging his time on the base._

--

Therefore, we need to implement this exception, because it is not connected now:

![Image](https://github.com/user-attachments/assets/8c8dea9a-579e-4d8f-b92d-33d5b06c6c82)

**Comment by AloisSeckar** [comment]
Implemented in fe7cde7ab0bec1b13f4b1c2596849a2f096a60b3
